src/main/java/tc/oc/pgm/commands/ModerationCommands.java

```diff
@@ -0,0 +1,70 @@
+package tc.oc.pgm.commands;
+
+import app.ashcon.intake.Command;
+import app.ashcon.intake.parametric.annotation.Text;
+import net.md_5.bungee.api.ChatColor;
+import org.bukkit.Bukkit;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+import tc.oc.component.Component;
+import tc.oc.component.types.PersonalizedText;
+import tc.oc.component.types.PersonalizedTranslatable;
+import tc.oc.named.NameStyle;
+import tc.oc.pgm.api.Permissions;
+import tc.oc.pgm.api.chat.Audience;
+import tc.oc.pgm.api.match.Match;
+import tc.oc.pgm.api.player.MatchPlayer;
+import tc.oc.pgm.events.PlayerReportEvent;
+
+public class ModerationCommands {
+
+  @Command(
+      aliases = {"report"},
+      usage = "<player> <reason>",
+      desc = "Report a player who is breaking the rules")
+  public static void report(
+      CommandSender commandSender,
+      MatchPlayer matchPlayer,
+      Match match,
+      Player player,
+      @Text String reason) {
+    MatchPlayer accused = match.getPlayer(player);
+    PlayerReportEvent event = new PlayerReportEvent(commandSender, accused, reason);
+    match.callEvent(event);
+
+    if (event.isCancelled()) {
+      if (event.getCancelMessage() != null) {
+        commandSender.sendMessage(event.getCancelMessage());
```

Comment:
`matchPlayer.sendMessage`

Reply:
The use of `CommandSender` allows for console reports. I can remove this feature if someone lets me know how to prevent Console from running the command as requested previously in a discussion with Molka.